### PR TITLE
Fix typo of window_state_context.js's path

### PR DESCRIPTION
**Pull request by makotokato**
I think that window_state_context.js path by https://github.com/web-platform-tests/wpt/pull/36734 is incorrect.

### CODE REVIEW

**Review by marcoscaceres — APPROVED** [review]
Ooops! Not sure how that happened. Thanks again @makotokato! 😅

**Comment by makotokato** [comment]
I have no write access permission for wpt repository, so could you merge this if no problem?

**Pull request merged**
